langchain/chat_models/anthropic.py

```diff
@@ -0,0 +1,148 @@
+from typing import List, Optional
+
+from pydantic import Extra
+from langchain.chat_models.base import BaseChatModel
+from langchain.llms.anthropic import _AnthropicCommon
+from langchain.schema import (
+    AIMessage,
+    BaseMessage,
+    ChatGeneration,
+    ChatMessage,
+    ChatResult,
+    HumanMessage,
+    SystemMessage,
+)
+
+
+class AnthropicChat(BaseChatModel, _AnthropicCommon):
+    r"""Wrapper around Anthropic's large language model.
+
+    To use, you should have the ``anthropic`` python package installed, and the
+    environment variable ``ANTHROPIC_API_KEY`` set with your API key, or pass
+    it as a named parameter to the constructor.
+
+    Example:
+        .. code-block:: python
+            import anthropic
```

Comment:
```suggestion
```

Reply:
This import is necessary for the use of `anthropic.HUMAN_PROMPT` down below.